src/components/Drawer/Drawer.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import { IoIosArrowUp } from 'react-icons/io';
+import styles from './Drawer.scss';
+
+const { block, elem } = bem({
+    name: 'Drawer',
+    classnames: styles,
+    propsToMods: ['isOpen'],
+});
+
+const Drawer = props => {
+    const { title, initialIsOpen, children, ...rest } = props;
+    const [isOpen, setIsOpen] = useState(initialIsOpen);
+
+    function handleClick() {
+        setIsOpen(!isOpen);
```

Comment:
In IE11 on browserstack Drawer doesn't collapse or expand. If I click on the `extendButton` – nothing happens :(

Reply:
Fixed, thanks!